src/UglyToad.PdfPig.Fonts/CompactFontFormat/Charsets/CompactFontFormatCharset.cs

```diff
@@ -37,7 +37,7 @@ public virtual string GetNameByGlyphId(int glyphId)
 
         public virtual string GetNameByStringId(int stringId)
         {
-            return GlyphIdToStringIdAndName.Single(x => x.Value.stringId == stringId).Value.name;
+            return GlyphIdToStringIdAndName.SingleOrDefault(x => x.Value.stringId == stringId).Value.name;
```

Comment:
Would this not throw where we access `.Value.name` or is the thing being accessed a tuple or struct anyway?